tigobot/scripts/pub.coffee

```diff
@@ -2,6 +2,6 @@
 #   Pub!
 
 module.exports = (robot) ->
-  robot.hear /pub/i, (msg) ->
+  robot.hear /^pub\??$/i, (msg) ->
```

Comment:
Could you change the regex to `/^pu+b\??$/i` to allow for such things as `<Norgg> puuuuub`